CHANGELOG.md

```diff
@@ -54,7 +56,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Corrected format of payment methods
 
-[Unreleased]: https://github.com/itk-dev/os2forms_selvbetjening/compare/1.1.1...HEAD
+[Unreleased]: https://github.com/itk-dev/os2forms_selvbetjening/compare/1.1.2...HEAD
```

Comment:
Hmm, it seems we have copy pasted these wrong since 1.0.0. They should say `os2forms_payment` not `os2forms_selvbetjening`